app/javascript/src/components/Contacts/contacts.js

```diff
@@ -0,0 +1,149 @@
+export const contacts = [
+  {
+    id: 1,
+    name: "Udai Gupta",
+    role: "Engineer",
+    email: "udai.gupta@bigbinary.com",
```

Comment:
Never use real world names or emails in seed data. Use random emails which ends in `example.com`.

Also in BigBinary we stick onto certain personas: https://handbook.neetokb.com/articles/personas